roles/openshift_facts/library/openshift_facts.py

```diff
@@ -717,7 +717,8 @@ def set_etcd_facts_if_unset(facts):
         try:
             # Add a fake section for parsing:
             ini_str = '[root]\n' + open('/etc/etcd/etcd.conf', 'r').read()
-            ini_fp = StringIO.StringIO(ini_str)
+            # pylint: disable=not-callable
+            ini_fp = StringIO(ini_str)
```

Comment:
Does the import for StringIO need to change? https://docs.python.org/2/library/stringio.html

I suspect the work in Ansible to support python3 may be coming into play here as well.